contrib/kube-prometheus/assets/prometheus/rules/kubelet.rules.yaml

```diff
@@ -18,20 +18,17 @@ groups:
     labels:
       severity: critical
     annotations:
-      description: '{{ $value }} Kubernetes nodes (more than 10% are in the NotReady
-        state).'
-      summary: Many Kubernetes nodes are Not Ready
+      description: '{{ $value }}% of Kubernetes nodes are not ready'
   - alert: K8SKubeletDown
-    expr: count(up{job="kubelet"} == 0) / count(up{job="kubelet"}) > 0.03
+    expr: count(up{job="kubelet"} == 0) / count(up{job="kubelet"}) * 100 > 3
     for: 1h
     labels:
       severity: warning
     annotations:
       description: Prometheus failed to scrape {{ $value }}% of kubelets.
-      summary: Many Kubelets cannot be scraped
   - alert: K8SKubeletDown
-    expr: absent(up{job="kubelet"} == 1) or count(up{job="kubelet"} == 0) / count(up{job="kubelet"})
-      > 0.1
+    expr: (absent(up{job="kubelet"} == 1) or count(up{job="kubelet"} == 0) / count(up{job="kubelet"}))
+      * 100 > 1
```

Comment:
In previous commit, it check 10% over of `up{job="kubelet"} == 0`.
So, `expr` is below, I think.

```diff
-      * 100 > 1
+      * 100 > 10
```

If it's my misunderstanding, sorry for this comment.

Reply:
@brancz Yes, I've create PR #1032 .